my_list.rb

```diff
@@ -0,0 +1,20 @@
+require './my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
+
```

Comment:
- [ ] It would be better if `MyList` implements #each method that yields all members of @list in succession.

![Screenshot from 2022-05-05 15-52-19](https://user-images.githubusercontent.com/64173189/167023626-d245117c-3960-49ae-91a6-94ea9a13ccc0.png)